frontend/http/writer_test.go

```diff
@@ -1,15 +1,12 @@
-// Copyright 2016 The Chihaya Authors. All rights reserved.
-// Use of this source code is governed by the BSD 2-Clause license,
-// which can be found in the LICENSE file.
-
 package http
 
 import (
 	"net/http/httptest"
 	"testing"
 
-	"github.com/chihaya/chihaya/tracker"
 	"github.com/stretchr/testify/assert"
```

Comment:
Yea, it's up to you. I don't really care too much, since all it does is eliminate a small amount of boilerplate.